backend/app/views/spree/admin/promotions/_form.html.erb

```diff
@@ -1,75 +1,59 @@
-          <%= bf.text_field :number_of_codes, :readonly => !@promotion.new_record?, class: 'fullwidth' %>
-        <% end %>
-      <% end %>
-
-      <%= f.field_container :per_code_usage_limit do %>
-        <%= f.label :per_code_usage_limit %>
-        <%= f.text_field :per_code_usage_limit, class: 'fullwidth' %>
-      <% end %>
-
-      <%= f.field_container :path do %>
-        <%= f.label :path %>
-        <%= f.text_field :path, :class => 'fullwidth' %>
-      <% end %>
-
-      <%= f.field_container :advertise do %>
-        <%= f.check_box :advertise %>
-        <%= f.label :advertise %>
-      <% end %>
-
-      <%= f.field_container :apply_automatically do %>
-        <%= f.check_box :apply_automatically %>
-        <%= f.label :apply_automatically %>
-      <% end %>
+<fieldset class="form-group no-border-bottom">
+  <legend>General</legend>
+
+  <%= f.field_container :name do %>
+    <%= f.label :name %>
+    <%= f.text_field :name, :class => 'fullwidth' %>
+  <% end %>
+
+  <%= f.field_container :description do %>
+    <%= f.label :description %><br />
+    <%= f.text_area :description, :rows => 3, :class => 'fullwidth' %>
+  <% end %>
+
+  <%= f.field_container :category do %>
+    <%= f.label :promotion_category_id, Spree::PromotionCategory.model_name.human %><br />
+    <%= f.collection_select(:promotion_category_id, @promotion_categories, :id, :name, { :include_blank => Spree.t('match_choices.none') }, { :class => 'select2 fullwidth' }) %>
+  <% end %>
+
+  <%= f.field_container :advertise do %>
+    <%= f.check_box :advertise %>
+    <%= f.label :advertise %>
+  <% end %>
+</fieldset>
+
+<fieldset class="form-group no-border-bottom">
+  <legend>Limits</legend>
+
+  <div class="row">
+    <div class="col-xs-6">
+      <div id="starts_at_field" class="field">
+        <%= f.label :starts_at %>
+        <%= f.text_field :starts_at, :value => datepicker_field_value(@promotion.starts_at), :class => 'datepicker datepicker-from fullwidth' %>
+      </div>
     </div>
 
-    <div class="omega nine columns">
-      <%= f.field_container :description do %>
-        <%= f.label :description %><br />
-        <%= f.text_area :description, :rows => 7, :class => 'fullwidth' %>
-      <% end %>
-
-      <%= f.field_container :category do %>
-        <%= f.label :promotion_category_id, Spree::PromotionCategory.model_name.human %><br />
-        <%= f.collection_select(:promotion_category_id, @promotion_categories, :id, :name, { :include_blank => Spree.t('match_choices.none') }, { :class => 'select2 fullwidth' }) %>
-      <% end %>
+    <div class="col-xs-6">
+      <div id="expires_at_field" class="field">
+        <%= f.label :expires_at %>
+        <%= f.text_field :expires_at, :value => datepicker_field_value(@promotion.expires_at), :class => 'datepicker datepicker-to fullwidth' %>
+      </div>
     </div>
   </div>
 
-  <div id="expiry_fields" class="three columns omega">
-    <%= f.field_container :overall_usage_limit do %>
```

Comment:
This is a super-nit. Should we be mixing old- and new-style Hash syntax in the same line? I feel like if we touch the line anyways, we could just switch it over to new if nothing stands against it.